src/index.ts

```diff
@@ -1,34 +1,44 @@
 import { Controller } from 'stimulus'
-import formatDistanceToNow from 'date-fns/formatDistanceToNow'
+import { formatDistanceToNow } from 'date-fns'
 
 export default class extends Controller {
+  isValid: boolean
+  refreshTimer: number
+  locale: object
+
+  hasRefreshIntervalValue: boolean
```

Comment:
I don't see a declaration of this value in static values,

hasAddSuffixValue and hasIncludeSecondsValue.

As far as I have used stimulus all values must be declared in static values ={}, because it is a kind of constructor https://stimulus.hotwire.dev/reference/values